diff --git a/esm/src/errors.js b/esm/src/errors.js
index 15ebe973ec2f98600798968cb4f782fa9d7aae02..34610c230a9011c5217b067e1c499f1eec1574a8 100644
--- a/esm/src/errors.js
+++ b/esm/src/errors.js
@@ -11,6 +11,9 @@ export class TemplateError extends VentoBaseError {
         this.path = path;
         this.source = source;
         this.position = position;
+				if (cause) {
+					this.message += `(via ${cause.name})\n`;
+				}
     }
 }
 export class TransformError extends VentoBaseError {
